sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -66,10 +66,10 @@ export interface CreateEcKeyOptions extends CreateKeyOptions {
 
 // @public
 export interface CreateKeyOptions extends coreHttp.OperationOptions {
-    curve?: KeyCurveName;
+    curveName?: KeyCurveName;
```

Comment:
Didn't you already ship with these names? Pretty sure you did, which means you can't change them now. They've been around for a while - `curveName` since GA.